docs/install/collecting-metrics/README.md

```diff
@@ -0,0 +1,101 @@
+This document describes how to set up the
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/about/) to
+receive metrics from the Knative infrastructure components and distribute them
+to Prometheus. [OpenTelemetry](https://opentelemetry.io/) is a CNCF project to
+provide consistent interfaces for instrumenting applications and platforms to
+provide telemetry information for developers and operators. OpenTelemetry allows
+Knative to build provider-agnostic instrumentation into the platform, so that
+it's easy to export logs to multiple monitoring services without needing to
+rebuild or reconfigure the Knative binaries.
+
+## Setting up the collector
+
+The collector provides a long-lived location where various Knative components
+can push metrics (and eventually traces) to be retained and collected by a
+monitoring service. For this example, we'll configure a single collector
+instance using the
+[opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-operator).
+
+![Diagram of components reporting to collector, which is scrapde by Prometheus](./system-diagram.svg)
+
+<!-- yuml.me UML rendering of:
+[queue-proxy1]->[Collector]
+[queue-proxy2]->[Collector]
+[autoscaler]->[Collector]
+[Collector]<-scrape[Prometheus]
+-->
+
+1. The first step is to install the operator:
+
+   ```shell
+   kubectl apply --filename https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
+   ```
+
+2. Next, create a namespace for the collector to run in:
+
+   ```shell
+   kubectl create namespace metrics
+   ```
+
+3. And then create an OpenTelemetryCollector object to store the configuration
+   for the collector:
+
+   ```shell
+   kubectl apply --filename collector.yaml
```

Comment:
We usually instruct in pre-reqs to git clone the repo and change directory, if not it will be a file not found

Reply:
I'm wondering about linking to `raw.githubusercontent.com` instead, so users don't need to do a clone. What do you think?

Reply:
Yeah I think is better to provide htttp URL to `raw.githubusercontent.com`